source/RoslynAnalyzers/AsyncAnalyzer.cs

```diff
@@ -32,22 +30,15 @@ void CheckNode(SyntaxNodeAnalysisContext context)
         var isAsync = methodDec.Modifiers.Any(SyntaxKind.AsyncKeyword);
         if (isAsync)
         {
-            VoidMethods_MustNotBeAsync(context, methodDec);
```

Comment:
I thought we were talking about removing AsyncFixer as it was a dependency?

Reply:
We were, but then Pawel thought one of the things we would lose would actually be quite useful, and Joel found the perf problems went away if we updated to the latest version, so we decided to keep it

https://octopusdeploy.slack.com/archives/C01HZFJRYSH/p1671052222784489

Reply:
Thanks for the context.